scripts/ubuntu-20-provision.sh

```diff
@@ -61,6 +61,14 @@ apt-get install -y --no-install-recommends \
   jq \
   parallel
 
+# setup qemu
+apt-get install -y --no-install-recommends \
+  qemu \
+  binfmt-support \
+  qemu-user-static
+
+docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
```

Comment:
@timja shouldn't we run this command in the pipelines, to have Qemu loaded only when we need it?

Reply:
It just configures qemu, what would the reason of moving it be?

Reply:
I'm not comletely sure that the setting is persisted across reboot: let's check as it does not create any issue (I was overcomplicating)